woocommerce/woocommerce-gutenberg-products-block
# Adds product query support for Category list block

Adds Product Query support for the atomic category list block.

On the client side, when the Category list Block is used within the Product Query block, the markup will be rendered on the server side - no javascript related to the Category list block will be loaded.





Fixes woocommerce/woocommerce-blocks#7329 




### Testing
1. Checkout this branch and run `npm start` 
2. Be sure that you have latest version of the Gutenberg feature plugin installed and activated
3. Create a post/page and add the Product Query block.
4. Add `woocommerce/product-category-list` block to `Post Template` of product query block
    - Verify that product category block works as it was working before.  
6. Now save the post & Open the frontend side and confirm that:
    - No JS related to the Product Category list block is loaded (search in your network tab for `product-category-list-frontend.js`).
    - The UI of the block is identical to on the editor side.
    - That the settings are reflected correctly (eg: custom style).
7. For comparison, switch to trunk and add the All Products block to confirm that the aforementioned product-category-list-frontend.js is loaded.


### WooCommerce Visibility
